conn.c

```diff
@@ -33,13 +33,13 @@ make_conn(int fd, char start_state, tube use, tube watch)
     job j;
     Conn *c;
 
-    c = new(Conn);
-    if (!c) return twarn("OOM"), NULL;
+    c = (Conn *)new(Conn);
```

Comment:
This is the error returned without these changes:
![beanstalkd beanstalkd-build 1 2016-04-03 21-27-43](https://cloud.githubusercontent.com/assets/1804/14237573/12dbef30-f9e3-11e5-8e42-17c20b590c1f.png)
This is with GCC 4.7.4 on SmartOS joyent_20160317T000621Z.